lfs.c

```diff
@@ -3761,6 +3761,13 @@ static int lfs_rawmount(lfs_t *lfs, const struct lfs_config *cfg) {
 
                 lfs->attr_max = superblock.attr_max;
             }
+
+            if ((superblock.block_count != lfs->cfg->block_count) ||
+                (superblock.block_size != lfs->cfg->block_size))
+            {
```

Comment:
nit picking: brace should be on previous line for consistency of code base.

Reply:
Agreed. Fixed below. I'm not sure how well github works with squashing commits. Let me know if there's anything I should do for that.

Reply:
looks good!

@maintainer let's approve and merge this :)